src/main/resources/messages_zh_CN.properties

```diff
@@ -808,9 +808,9 @@ pdfOrganiser.placeholder=（例如 1,3,2 或 4-8,2,10-12 或 2n-1）
 
 
 #multiTool
-multiTool.title=PDF多功能工具
-multiTool.header=PDF多功能工具
-multiTool.uploadPrompts=File Name
+multiTool.title=PDF多页面工具
```

Comment:
The original text means **multifunction PDF tool** in Chinese, I changed the text to:  **multi-page PDF tool** based on the feature, if it's not what we want, I'll revert this change.

Reply:
It's intended as a tool which can do page movements rotate and deletion etc so we currently call it multi tool
Honestly not happy with the name but that's what it currently is

Reply:
Got it, reverted the change for `multiTool`